lxljs/string.js

```diff
@@ -308,3 +308,16 @@ export function getFormattedEntries(list, vocab, language, context) {
   remove(formatted, value => value === '' || value === null); // Remove empty strings
   return formatted;
 }
+
+export function getSigelLabel(sigel, len) {
```

Comment:
It is mostly a hygiene detail. Is it possible to use a bit more generic or less libris-centric names excluding the word sigel in some of these function or var/const names? Since the sigel is by our definition the code on something and not the actual entity it describes (contrary to how we use it in natural language, the hard part here being it plays different roles and is not always a library). Are we meaning userLocation? Not a stopper by any means and should it turn out to be too timeconsuming to figure out I wouldn't argue it easier to revisit later when the model and definitions around this is clearer.

Reply:
We can change it to 'collection', because that's what it is called in the user store?